services/hosted_services.md

```diff
@@ -1,19 +1,95 @@
+
+### Web Interface
+Atlas hosts the web interface code in a [Github repo](https://github.com/AtlasSystems/hosting-aeon-princeton). You need to request access to this via Atlas support. [Documentation](https://support.atlas-sys.com/hc/en-us/articles/4407504126611-Editing-Atlas-Hosted-Aeon-Web-Pages-in-GitHub) on the workflow to allow PUL users to submit updates is available. The web interface is integrated via Shibboleth for Princeton net ID holders and offers a self-managed username and password for SC users outside of Princeton who want to request SC items. 
+
+## Staff Clients
+Staff clients are part of a Windows image. These are typically updated when Aeon has a new release or Atlas releases a security patch for Aeon. Staff users have a unique username/password stored in Aeon itself that they use to sign to the client. 
+
+* The "Aeon" staff client allows staff to process requests. 
+* "Staff Manager" allows you to create and manage existing staff client users. Including reseting passwords. 
+* "Customization Manager" contains configuration settings for Aeon as well as configuration settings for server-side add-ons.
+
+## Printing
+Aeon staff clients print out various labels that are used in the Reading Room circulation process. A description is available in the [Aeon print documentation](https://support.atlas-sys.com/hc/en-us/articles/360011920833-Aeon-Default-Print-Templates). All clients have a path that they resolve set in the Aeon Customization Manager key "PrintDocumentsPath". Currently that is set to a file share on the lib-aeon.princeton.edu server. 
+
+## Add-ons
+Aeon has an add-on architecture implemented in [LUA](https://www.lua.org/). Add-ons can be added at the individual client level or on the server. Client add-ons are incorporated into our Aeon client install in our staff workstation image and replicated on every client instance. Server add-ons are installed in a single place within the Aeon server application. See available [add-ons](https://atlas-sys.atlassian.net/wiki/spaces/ILLiadAddons/pages/3149603/Aeon+Addon+Directory).
+
+### Server Add-ons we use.
+* [ReCAP Add-On](https://github.com/PrincetonUniversityLibrary/aeon_scsb_addon) - Facilitates transactions with SCSB
+
+### Getting Support 
+General documentation is available on the [Atlas website](https://support.atlas-sys.com/hc/en-us/categories/360000720853-Aeon). Support can be requested through the [Atlas Web Portal](https://support.atlas-sys.com/hc/en-us/requests/new)
+
+## Using the client for troubleshooting/account management
+The Aeon client can only be reached from PUL Windows machines. IT staff who do not regularly work on Windows machines must use Windows Remote Desktop to connect to one of the desktop Windows machines listed below. Once connected, staff must use actual Aeon client staff credentials to login into the various clients. 
+
+* lib-ares-cli1
+* lib-ares-cli2
+* lib-ares-cli3
```

Comment:
Do we use the same three machines for both Aeon and Ares???

```suggestion
* lib-aeon-cli1
* lib-aeon-cli2
* lib-aeon-cli3
```

Reply:
Yes, we use the same machines with all three atlas applications.